### ISSUE TITLE
marching_cubes with color from array

### ISSUE THREAD

**Issue by sTamaroTum** [K3D-tools/K3D-jupyter#427]
Hi, is it possible to give a color to the iso surfaces that come out of k3d.marching_cubes(A) based on an array with the same dimensions as A? Right now I could only set a uniform color. Thanks.

**Comment by artur-trzesiok** [comment]
I like that idea :)

**Comment by artur-trzesiok** [comment]
![image](https://github.com/K3D-tools/K3D-jupyter/assets/5256837/91bb8d32-c20e-4104-ae58-090cf3ef5b30)
Will be available in next release :) - thanks for inspiration!

**Comment by artur-trzesiok** [comment]
example is on devel: https://github.com/K3D-tools/K3D-jupyter/blob/devel/examples/marchingcubes_with_attribute.ipynb

**Comment by sTamaroTum** [comment]
Hi! Thanks a lot for implementing this, it is a nice feature. When I tried it I was a bit confused, since the velocity colors seem to be on different coordinates than what I expected. 

To double-check, I tried marching cubes on an array with a certain level _x_, and then I asked for a color attribute using the same array. I would expect the color to be uniform and equal to _x_, but, it is not. Could this be? Thanks!
